WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -26,6 +26,37 @@ final class ShippingLabelFormViewModel {
     private(set) var destinationAddress: ShippingLabelAddress?
     private(set) var packagesResponse: ShippingLabelPackagesResponse?
     private(set) var selectedPackageID: String?
+    var selectedPackage: ShippingLabelPackageSelected? {
+        guard let packagesResponse = packagesResponse else {
+            return nil
+        }
+
+        for customPackage in packagesResponse.customPackages {
+            if customPackage.title == selectedPackageID {
+                return ShippingLabelPackageSelected(boxID: customPackage.title,
+                                                    length: customPackage.getLength(),
+                                                    width: customPackage.getWidth(),
+                                                    height: customPackage.getHeight(),
+                                                    weight: customPackage.getWidth(),
+                                                    isLetter: customPackage.isLetter)
+            }
+        }
```

Comment:
⚠️ I think we should use an `if let` here instead so it's easier to see that this is an early exit. 

```suggestion
        if let customPackage = packagesResponse.customPackages.first(where: { $0.title == selectedPackageID }) {
            return ShippingLabelPackageSelected(boxID: customPackage.title,
                                                length: customPackage.getLength(),
                                                width: customPackage.getWidth(),
                                                height: customPackage.getHeight(),
                                                weight: customPackage.getWidth(),
                                                isLetter: customPackage.isLetter)
        }

```

What do you think?